src/clr-icons/clr-icons.spec.ts

```diff
@@ -134,6 +149,15 @@ describe('ClarityIcons', () => {
       }).toThrowError(expectedErrorMessage);
     });
 
+    it('should cast name to lowerCase', () => {
+      const shapeName = 'nameWithCapitalLetter';
+      const shapeTemplate = '<svg><title>shape template</title></svg>';
+      const shape = { [shapeName]: shapeTemplate };
+
+      ClarityIcons.add(shape);
+      expect(ClarityIcons.get(shapeName)).toBe(shapeTemplate);
```

Comment:
Can we use the lower case version of the shapeName instead of passing the exactly same?
```
expect(ClarityIcons.get(nameWithCapitalLetter.toLowerCase())).toBe(shapeTemplate);
```

Reply:
Yes, but will add another test for that.

It has to match both `someUpperCaseName` and `someuppercasename` but at the same time don't force the developer to write it like one way only.

Also, I'm thinking of updating the docs about that..